web/src/main/java/org/cbioportal/genome_nexus/web/InfoController.java

```diff
@@ -0,0 +1,27 @@
+package org.cbioportal.genome_nexus.web;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.cbioportal.genome_nexus.web.config.PublicApi;
+import org.springframework.web.bind.annotation.*;
+import java.util.Collections;
+import java.util.Map;
+
+@RestController // shorthand for @Controller, @ResponseBody
+@CrossOrigin(origins="*") // allow all cross-domain requests
+@RequestMapping(value= "/")
+@Api(tags = "info-controller", description = "Info Controller")
+@PublicApi
+public class InfoController
+{
+    @ApiOperation(value = "Retrieve Genome Nexus Version",
+        nickname = "fetchVersionGET")
+    @RequestMapping(value = "/version",
+        method = RequestMethod.GET,
+        produces = "application/json")
+    @ResponseBody
+    public Map<String,String> fetchVersionGET()
```

Comment:
As far as I remember, returning a Map was causing errors for swagger API client generator.